test/backend/QueryScheduler.spec.js

```diff
@@ -53,11 +53,12 @@ describe('QueryScheduler', function() {
 
     it('executes a function on an interval', function () {
         const spy = sinon.spy();
-        const refreshInterval = 1; // second
+        const refreshInterval = 5; // 5 seconds
+        const cronInterval = `*/${refreshInterval} * * * * *`;
```

Comment:
we should keep the units for testing the old format (i.e. undefined `cronInterval`).

Reply:
The problem with keeping `refreshInterval` is that it is handled differently in the `queryScheduler.scheduleQuery` call. Currently, refresh intervals are mapped to their corresponding cron expression before being scheduled.  If they match one of the [supported values](https://github.com/plotly/falcon-sql-client/issues/425#issuecomment-403959976), they're converted accordingly, but if they're not recognized as a supported value (as is the case of the value passed in the test), they default to a schedule of every week.

But I can update the code to accept a small enough `refreshInterval` to test if you'd like? I had assumed you wouldn't want me to alter the desired logic of the scheduler solely to conform to an existing test, which is why I originally opted to update the test units instead.

Reply:
On second thought, I updated the tests to use fake timers which eliminates this issue. The original `refereshInterval` format has been restored.